### PR TITLE
Create CONTRIBUTING.rst and LICENSE

### LINKED ISSUE #34: Better expose documentation for file names, etc.
https://github.com/neuropoly/spinalcordtoolbox/wiki/variables#variables

we should create a contribution guidelines

### LINKED ISSUE #39: Add a license file to the project
## Context 
Our project does not have any license file yet.
Having a license file might for example be useful for someone who wants to contribute by adding some external features and needs to check if these features can be used under our license.

### PR DESCRIPTION
**Pull request by gaspardcereza**
## Context
The `contributing` file stands as a guideline for any new team member or external contributor that wants to help us with the project.
Closes #34 
Closes #39 

## About this pull request
I  created a CONTRIBUTING.RST file based on what had been done for spinal chord toolbox.

There are still one uncompleted section that we need to discuss: We currently have no license file for the project (see #39 )

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,263 @@
+Contributing to shimming-toolbox
+===================
```
Comment (jcohenadad) [inline CONTRIBUTING.rst:2]: same number of "=" as number of chars for headings

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+issue (easier to read than having to click on the link).
+
+If the PR fixes issue(s), indicate it after your introduction:
+``Fixes #XXXX, Fixes #YYYY``. Note: it is important to respect the
+syntax above so that the issue(s) will be closed upon merging the PR.
+
+PR Labels
+~~~~~~~~~
+
+Add the labels that best describe your PR ('enhancement', 'documentation', etc...
```
Comment (jcohenadad) [inline CONTRIBUTING.rst:226]: do it as done [here](https://github.com/neuropoly/spinalcordtoolbox/blob/master/CONTRIBUTING.rst#pr-labels).

which also means: please create appropriate labels categories

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+Only request a review when you deem the PR as "good to go". If the PR is
+not ready for review, convert it to a "Draft".
+
+Github may suggest you to add particular reviewers to your PR. If that's
+the case and you don't know better, add all of these suggestions. The
+reviewers will be notified when you add them.
+
+Versioning
+----------
+--> TODO
```
Comment (jcohenadad) [inline CONTRIBUTING.rst:263]: why not using [that](https://github.com/neuropoly/spinalcordtoolbox/blob/master/CONTRIBUTING.rst#versioning)?

Comment (gaspardcereza) [reply CONTRIBUTING.rst:263]: That's what I was thinking but I wanted to know if it was ok for you.

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -223,7 +223,14 @@ syntax above so that the issue(s) will be closed upon merging the PR.
 PR Labels
 ~~~~~~~~~
 
-Add the labels that best describe your PR ('enhancement', 'documentation', etc...
+You **must** add labels to your PRs, as these are used to automatically generate Changelog:
+
+- **Category:** Choose **one** label that describes the `category <https://github.com/shimming-toolbox/shimming-toolbox-py/wiki/Label-definition#Category>`_ (purple labels).
```
Comment (jcohenadad) [inline CONTRIBUTING.rst:228]: you can delete the wiki page that you've created to replace with this link (with every new wiki page, we need to maintain the documentation)
```suggestion
- **Category:** Choose **one** label that describes the `category <https://github.com/shimming-toolbox/shimming-toolbox-py/labels?q=bug+documentation+feature+enhancement+testing+installation>`_.
```

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -223,7 +223,14 @@ syntax above so that the issue(s) will be closed upon merging the PR.
...
+
+- **Shimming-toolbox Function:** Choose one or multiple labels corresponding to the files/functions that are mainly affected by the PR (light blue labels).
```
Comment (jcohenadad) [inline CONTRIBUTING.rst:230]: for now, i suggest we remove those sub-labels (it's not like we have 50+ functions to maintain)
```suggestion

```

**Comment by gaspardcereza** [comment]
@jcohenadad I copied the license file from the Matlab project and added a link to it. I think we're ready to merge.

**Review by jcohenadad — APPROVED** [review]
awesome 💯

**Pull request merged**